v2/types_gen.go

```diff
@@ -1,10 +1,11 @@
 package v2
 
-// TODO: go build will build the latest version of the protoc-gen-gofast module
-// used by the project. We need a way to pin the exact version of
-// protoc-gen-gofast used in this generator. We could not find a way of doing
-// this at the time of this writing.
-
+//
+//TODO: go build will build the latest version of the protoc-gen-gofast module
```

Comment:
Nitpick: I think your editor might be removing the spaces after `//` in comments. We probably want the spaces for consistency.